dist/linux/byedpi_system.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=ByeDPI
+Documentation=https://github.com/hufrea/byedpi
```

Comment:
After=network.target ?

Reply:
Makes sense, but again, it raises some questions. Should `Wants=network.target` also be specified. Or even:

```service
[Unit]
# ..
After=network-online.target
Wants=network-online.target
``` 
Because, `network.target` does not guarantee that the network is actually connected; it only indicates that the networking subsystem is available. All this is beyond my shallow knowledge of systemd.

Reply:
I looked at what the `.service` file looks like in AUR. And decided to add these fields after all. If anyone has any thoughts on this matter, I'm all ears.

Reply:
This forces service to be loaded after network setup. This can be pretty useful because even loopback interface does not have its address before initial network setup on linux. But most of the time it should just work without it at least on systemd. And i think check if system actually connected is not required. byedpi probably should not send any packet if you don't use it.